### PR TITLE
test: improve test coverage to 80%

### PR DESCRIPTION
**Pull request by JeyJeyGao**
Test:
- improved test coverage to over 80.16%

Fix:
- added ORAS copy library to copy the OCI Layout test data to a temporary directory and avoid generating test time temporary files in the repository directory.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,4 +18,7 @@
 *.sublime-workspace
 
 # Custom
-coverage.txt
\ No newline at end of file
+coverage.txt
+
+# tmp directory was generated by example_remoteVerify_test.go
```
Comment (shizhMSFT) [inline .gitignore:23]: I'm not able to find `example_remoteVerify_test.go` in this PR.

Comment (JeyJeyGao) [reply .gitignore:23]: https://github.com/notaryproject/notation-go/blob/5e98995bd17a4ed9ad435c7e6d240904d183ed56/example_remoteVerify_test.go#L127-L131

Every time the unit test runs locally, it generates a temporary file, which is not convenient.

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Review by Two-Hearts — APPROVED** [review]
LGTM

**Review by priteshbandi — APPROVED** [review]
LGTM

**Pull request merged**